python/packages/autogen-agentchat/src/autogen_agentchat/agents/_assistant_agent.py

```diff
@@ -181,9 +184,13 @@ def __init__(
         description: str = "An agent that provides assistance with ability to use tools.",
         system_message: str
         | None = "You are a helpful AI assistant. Solve tasks using your tools. Reply with TERMINATE when the task has been completed.",
+        max_tool_iterations: int = 1,
+        return_only_response: bool = False,
     ):
         super().__init__(name=name, description=description)
         self._model_client = model_client
+        self._max_tool_iterations = max_tool_iterations
```

Comment:
Check `max_tool_iterations` in constructor: it must be greater than or equal to 1.